src/components/Search/SearchListWithHeader.tsx

```diff
@@ -50,34 +51,36 @@ function SearchListWithHeader(
     const {translate} = useLocalize();
     const [isModalVisible, setIsModalVisible] = useState(false);
     const [longPressedItem, setLongPressedItem] = useState<TransactionListItemType | ReportListItemType | null>(null);
-    const [selectedItems, setSelectedItems] = useState<SelectedTransactions>({});
-    const [selectedItemsToDelete, setSelectedItemsToDelete] = useState<string[]>([]);
+    const [selectedTransactions, setSelectedTransactions] = useState<SelectedTransactions>({});
+    const [selectedReports, setSelectedReports] = useState<Array<SearchReport['reportID']>>([]);
+    const [selectedTransactionsToDelete, setSelectedTransactionsToDelete] = useState<string[]>([]);
     const [deleteExpensesConfirmModalVisible, setDeleteExpensesConfirmModalVisible] = useState(false);
+    const [offlineModalVisible, setOfflineModalVisible] = useState(false);
 
     const handleOnSelectDeleteOption = (itemsToDelete: string[]) => {
-        setSelectedItemsToDelete(itemsToDelete);
+        setSelectedTransactionsToDelete(itemsToDelete);
         setDeleteExpensesConfirmModalVisible(true);
     };
 
     const handleOnCancelConfirmModal = () => {
-        setSelectedItemsToDelete([]);
+        setSelectedTransactionsToDelete([]);
         setDeleteExpensesConfirmModalVisible(false);
     };
 
-    const clearSelectedItems = () => setSelectedItems({});
+    const clearSelectedTransactions = () => setSelectedTransactions({});
```

Comment:
We should clear the selected reports along with the transactions.

Reply:
Done